fluffy/templates/paste.html

```diff
@@ -31,13 +31,29 @@
 {% endblock %}
 
 {% block text %}
-    <div class="line-numbers">
-        {% for i in range(1, num_lines(text) + 1) %}
-            <a id="LL{{i}}">{{i}}</a>
-        {% endfor %}
-    </div>
-    <div class="text" contenteditable="true" spellcheck="false">
-        {{highlighter.highlight(text)|safe}}
+    <div style="display:flex;">
+        <div style="flex:1; width:30%;">
+            <div class="line-numbers" style=>
+                {% for i in range(1, num_lines(text) + 1) %}
+                    <a id="LL{{i}}">{{i}}</a>
+                {% endfor %}
+            </div>
+            <div class="text" contenteditable="true" spellcheck="false">
+                {{highlighter.highlight(text)|safe}}
+            </div>
+        </div>
+        <div style="flex:1; width:30%;">
+            {% if text2 is defined and highlighter2 is defined%}
+                <div class="line-numbers" style=>
+                    {% for i in range(1, num_lines(text2) + 1) %}
+                        <a id="LL{{i}}">{{i}}</a>
```

Comment:
This will repeat element IDs from the previous line numbers rendered which isn't valid HTML.


Maybe we should omit the IDs on the second set of line numbers, if the left and right views always have the same numbers anyway?